### DISCUSSION TITLE
How to keep controls visibility when BetterPlayerListVideoPlayer scrolling invisible inListView ?

### DISCUSSION THREAD

**Discussion by lycstar** [jhomlala/betterplayer#760] [Q&A]
How to keep controls always visibility when BetterPlayerListVideoPlayer scrolling invisible in ListView , In this way, users can click play again more quickly. but now controls will auto hide.  

I've read the doc,  I've only seen this method in BetterPlayerController not BetterPlayerListVideoPlayerController. 
Thank you!

**Comment by jhomlala** [comment]
You can set `BetterPlayerController` instance in `BetterPlayerListVideoPlayerController.setBetterPlayerController` and you will be able to control visibility of the player controls.
